front/lib/upsert_queue.ts

```diff
@@ -46,8 +41,9 @@ export const EnqueueUpsertTable = t.type({
   tableParentId: t.union([t.string, t.undefined, t.null]),
   tableParents: t.union([t.array(t.string), t.undefined, t.null]),
   csv: t.union([t.string, t.null]),
+  // TODO(spolu): [CSV-FILE] Remove undefined once deployed
+  fileId: t.union([t.string, t.null, t.undefined]),
```

Comment:
fileId still allow undefined so won't break the ones on GCS not yet processed that don't have it